### DISCUSSION TITLE
Unable to perform DIPRO calculation

### DISCUSSION THREAD

**Discussion by Quark195** [grimme-lab/xtb#943] [Q&A]
Dear XTB developers!

I'm trying to perform a DIPRO calculation according to the manual, using the --dipro keyword. However, the program displays a warning: 

> "[WARNING] Please study the warnings concerning your input carefully -1- prog_main_parseArguments: Unknown option '--dipro' provided"<

I used the latest developer version 6.6.1, as well as the most recent stable version 6.6.1.

Please clarify how to properly run the DIPRO calculation in the current program version.

I appreciate your help in advance.

**Comment by Albkat** [comment]
Hello,  DIPRO is a new feature that would be integrated in the xtb 6.7.0. You could wait till the release or use the [bleed version](https://github.com/grimme-lab/xtb/releases/tag/bleed) .
